src/Evolution/DgSubcell/Tags/Interpolators.hpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <boost/functional/hash.hpp>
+#include <cstddef>
+#include <optional>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/FixedHashMap.hpp"
+#include "Domain/Structure/Direction.hpp"
```

Comment:
aren't forward declarations sufficient?

Reply:
A lot of these will be default initialized into the DataBox, so I don't know where I would put the includes otherwise.

Reply:
you can leave for now if you want;  I think we once we put all simple/base tags into tags directories, tag files shouldn't include headers whenever possible.  It's whatever uses the tags that should include them.

Reply:
In that case I'll switch to forward decls here (just so you don't merge too quickly :P )

Reply:
okay, done & pushed